README.md

```diff
@@ -369,24 +369,9 @@ In most cases you will not need to set a value for `responseMode`. Defaults are
-* `sameSite`: Defaults to `none` if the `secure` option is `true`, or `lax` if the `secure` option is false. Allows fine-grained control over the same-site cookie setting. A value of `none` allows embedding within an iframe. A value of `lax` will avoid being blocked by user "3rd party" cookie settings. A value of `strict` will block all cookies when redirecting from Okta and is not recommended.
+| `tokenManager` | An object containing additional properties used to configure the internal token manager. <ul><li>`autoRenew`: By default, the library will attempt to renew tokens before they expire. If you wish to  to disable auto renewal of tokens, set autoRenew to false.</li><li>`storage`: You may pass an object or a string. If passing an object, it should meet the requirements of a [custom storage provider](#storage). Pass a string to specify one of the built-in storage types:<ul><li>[`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) (default)</li><li>[`sessionStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage)</li><li>[`cookie`](https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie)</li><li>`memory`: a simple in-memory storage provider</li></ul></li><li>`storageKey`: By default all tokens will be stored under the key `okta-token-storage`. You may want to change this if you have multiple apps running on a single domain which share the same storage type. Giving each app a unique storage key will prevent them from reading or writing each other's token values.</li></ul> |
+| `cookies` | An object containing additional properties used when setting cookies. <ul><li>`secure`: Defaults to `true`, unless the application origin is `http://localhost`, in which case it is forced to `false`. If `true`, the SDK will set the "Secure" option on all cookies. When this option is `true`, an exception will be thrown if the application origin is not using the HTTPS protocol. Setting to `false` will allow setting cookies on an HTTP origin, but is not recommended for production applications.</li><li>`sameSite`: Defaults to `none` if the `secure` option is `true`, or `lax` if the `secure` option is false. Allows fine-grained control over the same-site cookie setting. A value of `none` allows embedding within an iframe. A value of `lax` will avoid being blocked by user "3rd party" cookie settings. A value of `strict` will block all cookies when redirecting from Okta and is not recommended.</li></ul>|
+| `storageKeys` | An object containing custom storage keys used to provide consistent storage keys for internal state managers. <ul><li>`idToken`: customized storage key for IDToken, default to `idToken`.</li><li>`accessToken`: customized storage key for AccessToken, default to `accessToken`.</li></ul> |
```

Comment:
Should we mention that we don't recommend changing storageKeys and just rely on storageKey unless necessary for compatibility with something else?  I think it's easy for people to make a mess without intending to, particularly since the names are so similar.

Also, do we have options to make the names less similar?  Perhaps 'storageKey' and 'tokenKeys'?

Reply:
After more review, this looks to be a breaking change.  Anything that relied on the previous token keys will break.

Reply:
hmm, with the options still default to the internal constants (https://github.com/okta/okta-auth-js/pull/462/files#diff-75082c639aefcc47b3b8df57c8870867R105), I don't see why it's a breaking change.
The problem we want to solve here is users may call `auth.tokenManager.add(key, token)` to add tokens to storage, but the key they provide may differ from what the sdk has in constants. Then `auth-js` need a way to know what's the keys users introduced. 
In current downstream SDKs, it might be a bug if the code stick with the constants but different keys are provided by users.

> Should we mention that we don't recommend changing storageKeys and just rely on storageKey unless necessary for compatibility with something else?

I don't see much problem as long as internal hardcode usage is properly handled. Users can ignore this option in most scenarios.

> Also, do we have options to make the names less similar? Perhaps 'storageKey' and 'tokenKeys'?

I sort of prefer the key-value pairs way of managing simple data structures. It's flexible to maintain w/o introduce too many options to the configuration.